examples/heat_equation.cpp

```diff
@@ -65,13 +65,19 @@ void display(double time, ChunkType temp)
                 std::cout << std::setw(6) << temp_slice(ix);
             });
     std::cout << " }" << std::endl;
+
+    ddc::PdiEvent("display")
+            .with("temp", temp)
+            .and_with("mean_temp", mean_temp)
+            .and_with("temp_slice", temp_slice);
 }
 //! [display]
 
 
 //! [main-start]
 int main(int argc, char** argv)
 {
+    PDI_init(PC_parse_string(""));
```

Comment:
Does it leak when the string is empty ?

Reply:
Yes, but is a leak that happens only once really a leak? :)

Reply:
I agree maybe not a true leak but still, it triggers the sanitizers. For this reason I would be in favor of calling the destructor of the object. It also shows the correct usage of the paraconf library.